src/PortlandLabs/ConcreteCmsTheme/Navigation/HeaderNavigationFactory.php

```diff
@@ -23,36 +23,69 @@ class HeaderNavigationFactory implements ApplicationAwareInterface, NavigationFa
     public function createNavigation(): NavigationInterface
     {
         $activeSection = $this->activeSection;
+        // Determine if this navigation menu is for the concretecms.org site or not
+        $site_target = $_ENV['URL_SITE_MARKETING_OPENSOURCE'];
+        $site_detected = \Core::make('site')->getSite()->getSiteCanonicalURL();
+        // remove trailing slash if exists
+        if (strcmp('/', substr($site_detected, -1)) === 0 ) {
+            $site_detected = substr($site_detected, 0, -1);
+        }
+        $site_is_marketing_opensource = false;
+        if (strcmp($site_target, $site_detected) === 0) {
+            $site_is_marketing_opensource = true;
+        }
 
         $navigation = new Navigation();
-        $navigation->add(new Item('{{marketing}}/about', t('About'), false, false, [
-            new Item('{{marketing}}/about/features', t('Features')),
-            new Item('{{marketing}}/about/case-studies', t('Case Studies')),
-            new Item('{{marketing}}/about/solutions', t('Solutions')),
-            new Item('{{marketing}}/about/blog', t('Blog')),
-            new Item('{{marketing}}/about/contact-us', t('Contact')),
-        ]));
+        // About
+        $navigation_about_menu = [];
+        if ($site_is_marketing_opensource) {
+            // About menu for concretecms.org
+            $navigation_about_menu = [
+                new Item('{{marketing_org}}/vision',                   t('Vision')),
```

Comment:
```
 44 | ERROR   | [x] Expected 1 space after comma in argument list; 19 found
    |         |     (Generic.Functions.FunctionCallArgumentSpacing.TooMuchSpaceAfterComma)
```

Also for the lines with the same matching indentation 45, 47, 48, 49, 50, 55, 56, 58, 60, 61, 67, 68, 74, 75, 76, 80, 81, 82, 87, 88